app/views/categories/_child_categories.html.erb

```diff
@@ -1,17 +1,26 @@
 <% contents.select(&:category?).each_with_index do |child_category, i| %>
-  <section class="category-detail">
-    <%= heading_tag child_category.title,
-                    level: 2,
+  <section class="category-detail" data-dough-component="Collapsable">
+    <%= heading_tag level: 2,
                     id:    child_category.id,
-                    class: 'category-detail__heading' %>
-
+                    class: 'category-detail__heading',
+                    data: { dough_collapsable_trigger: 'category_' + child_category.id } do
+    %>
+      <%= render 'shared/svg/use_icon', icon: 'plus', variant: 'yellow' %>
+      <%= render 'shared/svg/use_icon', icon: 'minus', variant: 'yellow' %>
+      <%= child_category.title %>
+    <% end %>
     <p class="category-detail__intro"><%= child_category.description %></p>
-
+    <%= render 'category_promos', promos: child_category.category_promos unless !child_category.category_promos || child_category.category_promos.empty? %>
```

Comment:
This could probably be put more concisely, but without knowing what kind of object `child_category` is it's hard to say. Could be `render [...] if child_category.category_promos.present?`